ui/src/environments/environment.test.ts

```diff
@@ -5,4 +5,5 @@ export const environment: Environment = {
   ...testEnvironmentBase,
   displayTag: 'Test',
   debug: false,
+  enablePublishedWorkspaces: true,
```

Comment:
nitpick: you should be able to set this to "true" in test-env-base and leave it out of this file. The test-env-base pattern was my (rather weak) attempt to reduce config duplication across the environments which are "test-like". However, I'm realizing this hasn't really caught on, and having each environment config be fully-specified may be the better path forward. Open to thoughts here (I had a round of comments w/ Andrew on a similar issue the other day... if you two decide this pattern isn't useful, please feel free to follow-up and remove it!)